frontend/src/lib/components/AnnotationsOverlay/annotationsOverlayLogic.test.ts

```diff
@@ -546,14 +584,15 @@ describe('annotationsOverlayLogic', () => {
                     logic = annotationsOverlayLogic({
                         dashboardItemId: MOCK_INSIGHT_SHORT_ID,
                         insightNumericId: MOCK_INSIGHT_NUMERIC_ID,
+                        dashboardId: MOCK_DASHBOARD_ID,
                         dates: ['2022-01-01', '2023-01-01'],
                         ticks: [{ value: 0 }, { value: 1 }],
                     })
                     logic.mount()
                     await expectLogic(annotationsModel).toDispatchActions(['loadAnnotationsSuccess'])
-                    await expectLogic(insightLogic({ dashboardItemId: MOCK_INSIGHT_SHORT_ID })).toDispatchActions([
-                        'loadInsightSuccess',
-                    ])
+                    await expectLogic(
+                        insightLogic({ dashboardItemId: MOCK_INSIGHT_SHORT_ID, dashboardId: MOCK_DASHBOARD_ID })
```

Comment:
Worth adding some cases for insights without `dashboardId` + insights with a _different_, irrelevant `dashboardId`